packages/website/pages/docs/guides/run-a-node/claim-prover-rewards.mdx

```diff
@@ -7,15 +7,22 @@ This guide will help you claim TTKO/TTKOe earned as a prover.
 ## Prerequisites
 
 - You followed the [Enable a prover](/docs/guides/enable-a-prover) guide and have a prover which has proved a block to earn TTKO/TTKOe rewards.
-- Must have a balance of ETH Sepolia (see: [Receive tokens](/docs/guides/receive-tokens)).
+
+### Claim TTKO
+
+- Must have a balance of ETH on Sepolia L1 (see: [Receive tokens](/docs/guides/receive-tokens)).
+
+### Claim TTKOe
+
+- Must have a balance of ETH on Grimsvotn L2 (see: [Receive tokens](/docs/guides/receive-tokens)/[Bridge tokens](/docs/guides/build-on-taiko/bridge-tokens)).
 
 ## Steps
 
 <Steps>
 
 ### View TTKO/TTKOe prover address balance
 
-Go to the TaikoL1 contract read page ([TTKO](https://sepolia.etherscan.io/address/0x6375394335f34848b850114b66A49D6F47f2cdA8#readProxyContract#F10)/[TTKOe](https://explorer.test.taiko.xyz/address/0x4e7c942D51d977459108bA497FDc71ae0Fc54a00/read-proxy#address-tabs)) and input your prover address to lookup your TTKO/TTKOe claim balance.
+Go to the TaikoL1/L2 contract read page ([TTKO](https://sepolia.etherscan.io/address/0x6375394335f34848b850114b66A49D6F47f2cdA8#readProxyContract#F10)/[TTKOe](https://explorer.test.taiko.xyz/address/0x4e7c942D51d977459108bA497FDc71ae0Fc54a00/read-proxy#address-tabs)) and input your prover address to lookup your TTKO/TTKOe claim balance.
```

Comment:
```suggestion
Go to the TaikoL1 contract read page ([TTKO](https://sepolia.etherscan.io/address/0x6375394335f34848b850114b66A49D6F47f2cdA8#readProxyContract#F10)/[TTKOe](https://explorer.test.taiko.xyz/address/0x4e7c942D51d977459108bA497FDc71ae0Fc54a00/read-proxy#address-tabs)) and input your prover address to lookup your TTKO/TTKOe claim balance.
```

Reply:
technically the contract name is TaikoL1 even though that is also deployed on Grimsvotn L2. we might consider renaming the contract to something like TaikoBase.